sos/report/plugins/ovn_central.py

```diff
@@ -131,10 +131,6 @@ def setup(self):
             'ovn-nbctl --no-leader-only show',
             'ovn-nbctl --no-leader-only get-ssl',
             'ovn-nbctl --no-leader-only get-connection',
-            'ovn-nbctl --no-leader-only list loadbalancer',
-            'ovn-nbctl --no-leader-only list Load_Balancer',
-            'ovn-nbctl --no-leader-only list ACL',
-            'ovn-nbctl --no-leader-only list Logical_Switch_Port',
```

Comment:
Those commands are not required at all, right?

Reply:
No, they are table names that appear in the schema analysis a few lines below. They were added a long time ago, when there was no such iteration over the schema table names.